server/src/feature_cache.rs

```diff
@@ -99,6 +99,7 @@ fn update_client_features(
             s
         }),
         query: old.query.clone().or(update.query.clone()),
+        meta: None,
```

Comment:
Should this be old.meta.clone()

Reply:
Hmm, probably `update.meta.clone().or(old.meta.clone())`

Reply:
Added